website/docs/r/serviceendpoint_sonarqube.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_serviceendpoint_sonarqube"
+description: |-
+  Manages a SonarQube server endpoint within Azure DevOps organization.
+---
+
+# azuredevops_serviceendpoint_sonarqube
+Manages a SonarQube service endpoint within Azure DevOps. 
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "project" {
+  name       = "Sample Project"
+  visibility         = "private"
+  version_control    = "Git"
+  work_item_template = "Agile"
+}
+
+resource "azuredevops_serviceendpoint_sonarqube" "serviceendpoint" {
+
+  project_id        = azuredevops_project.project.id
+  service_endpoint_name = "sonarqube-mainframe"
+  description           = "My Service Connection"
+  url                   = "https://sonarqube.my.com"
+  token                 = "REDACTEDREDACTED"
+}
```

Comment:
I would suggest change the example configuration to :
```suggestion
resource "azuredevops_serviceendpoint_sonarqube" "serviceendpoint" {

  project_id        = azuredevops_project.project.id
  service_endpoint_name = "Sample SonarQube"
  url                   = "https://sonarqube.my.com"
  token                 = "0000000000000000000000000000000000000000"
  description           = "Managed by Terraform"
}
```

Reply:
Done